packages/grafana-ui/src/components/Select/types.ts

```diff
@@ -77,6 +77,8 @@ export interface SelectCommonProps<T> {
   renderControl?: ControlComponent<T>;
   tabSelectsValue?: boolean;
   value?: T | SelectValue<T> | null;
+  /** Will wrap the MenuList in a react-window FixedSizeVirtualList for improved performance, does not support options with "description" properties */
+  virtualized?: boolean;
```

Comment:
Since this won't support options with descriptions I'm thinking that maybe it would be a good idea to expose this as its own Select component with a limited API instead of exporting it via the regular Select API (that currently is a mess).

Reply:
Good idea, I'll see if I can refactor this to be a separate `VirtualizedSelect` component 👍

Reply:
Let me know if you need any assistance with that.